<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Refresh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temperature">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_url">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>URL :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="url">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="lineWrapColumnOrWidth">
     <number>0</number>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;https://api.openweathermap.org/data/2.5/weather?q=clermont-ferrand&amp;amp;appid=9d76eced21e3494fa9c87561eed2c0aa&amp;amp;units=metric&amp;amp;lang=fr&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Ville_CB">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>clermont-ferrand</string>
    </property>
    <item>
     <property name="text">
      <string>clermont-ferrand</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>paris</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>londres</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>lyon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>marseille</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_ville">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Ville : </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="newVille">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="message">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>421</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ressenti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="prefix">
     <string>Température ressentie : </string>
    </property>
    <property name="suffix">
     <string>°</string>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="vitesse_vent">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="prefix">
     <string>vitesse du vent : </string>
    </property>
    <property name="suffix">
     <string>m/s</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="humidite">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="prefix">
     <string>humidité : </string>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="temps">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>741</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Temps : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dir_vent">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Direction du vent :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="direction_vent">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>269</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>url</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>url_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Ville_CB</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>CB_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Refresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newVille</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>new_ville()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>url_change()</slot>
  <slot>CB_change()</slot>
  <slot>search()</slot>
  <slot>new_ville()</slot>
 </slots>
</ui>
